h1st/model/fuzzy_logic/fuzzy_logic_model.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import Dict, NoReturn
+
+import skfuzzy
+from skfuzzy import control as ctrl
+
+from h1st.model.rule_based_model import RuleBasedModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyLogicModel(RuleBasedModel):
+    """
+    This FuzzyLogicModel is one kind of rule-based model. You can encode your
+    rules in a fuzzy way using variables with membership functions.
+    Add variables and use those variables when defining rules.
+
```

Comment:
This is better, but still somewhat obfuscated. Explain concisely something like, _FuzzyLogicModel is a RuleBasedModel. You need to things to create a FuzzyLogicModel: fuzzy variables, and fuzzy rules. Fuzzy variables are defined using membership functions or distributions. Fuzzy rules have predicates ... For more information, check out https://xxx _